download.html

```diff
@@ -28,9 +28,8 @@ <h1>Downloads</h1>
             <h2 id="swanlake">{{ site.data.swanlake-latest.metadata.version | capitalize_all }} <span id="versionInfo">({{ site.data.swanlake-latest.metadata.release-date | date: "%B %e, %Y" }})</span></h2>
             <div class="alert alert-info" style="border: 1px solid #20b6b0; background: #fff;" >
                <p>If you already have jBallerina installed, you can use the <a href="/learn/keeping-ballerina-up-to-date/">Ballerina Update Tool</a> to update it to {{ site.data.swanlake-latest.metadata.version | capitalize_all }}. To do this, first, execute the command below to get the Update Tool updated to its latest version. </p>
-               <pre>ballerina update</pre>
-               <p>From now onwards, the `ballerina` command has to be issued as `bal`.
-               Next, execute the command below to update to {{ site.data.swanlake-latest.metadata.version | capitalize_all }}.</p>
+               <pre>bal update</pre>
+               <p>If you are using an <strong>Update Tool version below 0.8.14</strong>, execute the `ballerina update` command to update it. Next, execute the command below to update to {{ site.data.swanlake-latest.metadata.version | capitalize_all }}.</p>
```

Comment:
Shouldn't this part come first? I mean, it'd be more likely that the users are coming from older versions of Ballerina given that we only released the version with the `bal` command change very recently right? @keizer619 thoughts?

Reply:
Lets include another section for `ballerina update` for users who using older version than 0.8.14

Reply:
@pubudu91